docs/content/how-to/node-tuning.md

```diff
@@ -95,4 +96,126 @@ If you would like to set some node-level tuning on the nodes in your hosted clus
     Example output:
     ```
     vm.dirty_ratio = 55
+    ```
+
+## Applying tuning which requires kernel boot parameters 
+You can also use the Node Tuning Operator for more complex tuning which requires setting kernel boot parameters. 
+As an example, the following steps can be followed to create a NodePool with huge pages reserved.
+
+1. Create the following ConfigMap which contains a Tuned object manifest for creating 10 hugepages of size 2M.
+
+    Save this ConfigMap manifest in a file called `tuned-hugepages.yaml`:
+    ```
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: tuned-hugepages
+      namespace: clusters
+    data:
+      tuned: |
+        apiVersion: tuned.openshift.io/v1
+        kind: Tuned
+        metadata:
+          name: hugepages
+          namespace: openshift-cluster-node-tuning-operator
+        spec:
+          profile:
+          - data: |
+              [main]
+              summary=Boot time configuration for hugepages
+              include=openshift-node
+              [bootloader]
+              cmdline_openshift_node_hugepages=hugepagesz=2M hugepages=50
+            name: openshift-node-hugepages
+          recommend:
+          - priority: 20
+            profile: openshift-node-hugepages
+    ```
+    > **_NOTE:_**  The `.spec.recommend.match` field is intentionally left blank. In this case this Tuned will be applied to all Nodes in the NodePool where this ConfigMap is referenced. It is advised to group Nodes with the same hardware configuration into the same NodePool. Not following this practice might result in TuneD operands calculating conflicting kernel parameters for two or more nodes sharing the same NodePool.
+
+    Create the ConfigMap in the management cluster:
+    ```
+    oc --kubeconfig="$MGMT_KUBECONFIG" create -f tuned-hugepages.yaml
+    ```
+
+2. Create a new NodePool manifest YAML file, customize the NodePools upgrade type, and reference the previously created ConfigMap in the `spec.tunedConfig` section before creating it.
```

Comment:
The example below clarifies this, but I believe we should clarify by:
`s/before creating it/before creating it in the management cluster/`.

Reply:
done